crates/frontend/src/pages/default_config/default_config.rs

```diff
@@ -35,7 +37,11 @@ pub fn default_config() -> impl IntoView {
 
     let selected_config = create_rw_signal::<Option<RowData>>(None);
     let key_prefix = create_rw_signal::<Option<String>>(None);
-    let enable_grouping = create_rw_signal(false);
+    let enable_grouping = create_rw_signal(
```

Comment:
```suggestion
    let enable_grouping =
        create_rw_signal(get_local_storage::<bool>("enable_grouping").unwrap_or(false));
```

Reply:
I think cargo-fmt does this

Reply:
@Datron , I was suggesting after making return type of get_local_storage generic, we can re-write that piece of code like this, you can see I added added `::<bool>`, which is suggesting the type we want.